packs/data/classfeatures.db/chirurgeon.json

```diff
@@ -21,7 +21,38 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.alchemist.fieldDiscovery",
+                "value": "Compendium.pf2e.classfeatures.qC0Iz6SlG2i9gv6g"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.alchemist.perpetualInfusions",
+                "value": "Compendium.pf2e.classfeatures.fzvIe6FwwCuIdnjX"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.alchemist.perpetualPotency",
+                "value": "Compendium.pf2e.classfeatures.VS5vkqUQu4n7E28Y"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.alchemist.greaterFieldDiscovery",
+                "value": "Compendium.pf2e.classfeatures.JJcaVijwRt9dsnac"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.alchemist.perpetualPerfection",
+                "value": "Compendium.pf2e.classfeatures.YByJ9O7oe8wxfbqs"
+            }
+        ],
```

Comment:
Instead of multiple AE-likes, have a single one that sets an object

Reply:
You blew my mind a little. I didn't realize I could pass objects with AE-likes.